### ISSUE TITLE
update README

### ISSUE THREAD

**Issue by hadifarnoud** [TheForgotten69/goinsta#10]
it looks like this fork is not going to be merged to main repo unfortunately. so it makes sense to update the README imho

**Comment by TheForgotten69** [comment]
Yep it will be done thanks to the work done in #8 in case you want to watch it!
